### ISSUE TITLE
Error/Crash when joining custom zombies servers.

### ISSUE THREAD

**Issue by espenshadow** [momo5502/boiii#381]
### Bug Type

Client

### BOIII Version

v0.0.1 (Latest)

### Environment (Operating System)

Microsoft Windows 10 (64-Bit)

### Reproducibility Rate

High: The bug can (almost) always be reproduced

### Steps to Reproduce

1. Launch boiii
2. Connect to server running any custom map (this case's map/gamemode is zm_der_riese [Der Riese Declassified])
3. Error/Crash

### Expected Result

Connect to the server and the map should start correctly.

### Actual Result

When attempting to join the server from the client with files present in usermap>workshopfoldername>zone we get the Error: Could not find rawfile "gamedata/constbaselines/zm/bl_little_endian_core_frontend_frontend.bin" / "Could not load level fastfile for level 'map_name' could not read fastfile (null)"

However the map can be played when simply starting a private match.

If the map files from the client are placed in >zone, the error is instead: exceeded: too many asset overrides in one call to DB_LoadXAssets.

### (Optional) Proof and Additional Information

usermaps>workshopfoldername>zone: https://imgur.com/zLa6TcM 
[boiii-crash-2023-03-22-16-20-20.zip](https://github.com/momo5502/boiii/files/11045551/boiii-crash-2023-03-22-16-20-20.zip)

...>zone: https://imgur.com/O6Ht5HG
[boiii-crash-2023-03-22-16-31-02.zip](https://github.com/momo5502/boiii/files/11045554/boiii-crash-2023-03-22-16-31-02.zip)

### Severity/Priority

Medium

**Comment by Brentdevent** [comment]
I can't replicate this issue with `zm_der_riese`, could you perhaps link the map here, so there is no mismatch between custom maps perhaps. I think I use the same map, I assume there is only one but I need to be sure.

![image](https://user-images.githubusercontent.com/70229620/227136809-e7e9467c-c3b3-4868-a281-ec813f1c576e.png)

Are you trying to load the custom map without steam?
If so the path should be `Call of Duty Black Ops III\usermaps\zm_der_riese\zone`  for example.

But this all seems like it could be broken files? How did you obtain these?

**Comment by espenshadow** [comment]
You are using the correct map (Der Riese: Declassified: https://steamcommunity.com/sharedfiles/filedetails/?id=2271346839) although the issue spans across six different maps tested.

We're running fitgirlv100 from 1337. I used steamcmd to download the files from the workshop. My file path matches the example path in client as well as server - I also attempted to join the zm_daybreak server that has been online for the last few days, which resulted in the same issue. To my knowledge, the issue is exclusive to my friends and I, so I now presume it's an issue with our installation of the game client itself and not boiii. We're going to reinstall the game files, use a fresh install of boiii, as well as the map, and grab the updated .exe from the cs forum. I'll send an update once all of that is complete.
